database.html

```diff
@@ -75,10 +73,78 @@ <h1>MyMood Database Table</h1>
       </ul>
     </div>
     
+    <!-- Zebra Striping -->
+    <script>
+    function zebraStripe() {
+      // Go through all rows that are visible, filter old rows by odd and even
+      var $allRows = $('tr:visible');
+      var $oddRows = $allRows.filter(':odd');
+      var $evenRows = $allRows.filter(':even');
+      
+      // Remove old classes, then add new ones
+      $oddRows.removeClass('even').addClass('odd');
+      $evenRows.removeClass('odd').addClass('even');
+    };
+    </script>
+    
     <!-- Search Function -->
     <script>
-      search.init(["#database_table"]);
-      search.searchRows();
+    function search() {
+      // Declare variables 
+      var input, filter, table, tr, searchOption, searchValue, td, i;
+      input = document.getElementById("searchInput");
+      filter = input.value.toUpperCase();
+      table = document.getElementById("database_table");
+      tr = table.getElementsByTagName("tr");
+      searchOption = document.getElementById("searchOption");
+      searchValue = searchOption.options[searchOption.selectedIndex].value;
+
+      // Loop through all table rows, and hide those who don't match the search query
+      for (i = 0; i < tr.length; i++) {
+        if (searchValue == "session_id") {
+          td = tr[i].getElementsByTagName("td")[0];
+        }
+        else if (searchValue == "user_id") {
+          td = tr[i].getElementsByTagName("td")[1];
+        }
+        else {
+          td = tr[i].getElementsByTagName("td")[2];
+        }
+        if (td) {
+          if (td.innerHTML.toUpperCase().indexOf(filter) > -1) {
+            tr[i].style.display = "";
+          } 
+          else {
+            tr[i].style.display = "none";
+          }
+        } 
+      }
+      zebraStripe();
+    }
+    </script>
+    
+    <!-- Read JSON File -->
+    <script>
+    $(document).ready(function readJson() {
+      $.getJSON("/assets/json/sessions.json", function(data) {
+        // Declare variables

```

Comment:
same as above

Reply:
I am assuming that the tr tag means tableRow, and then the td tag means tableData which represents a cell on the table. So then I'd be adding cells to the table row by row.